TeamCode/src/main/java/org/firstinspires/ftc/teamcode/robot/MecanumDriveSystem.java

```diff
@@ -83,17 +102,35 @@ public double ticksToRevolutions(int ticks)
     {
         return ((double) ticks / this.ticksPerRotation);
     }
+    
+    public int inchesToTicks(double inches)
+    {
+        return (int) Math.round(inches * this.ticksPerInch);
+    }
+
+    public double ticksToInches(int ticks)
+    {
+        return ((double) ticks / this.ticksPerInch);
+    }
 
     public int getMinimumDistanceFromTarget()
     {
         int d = this.motorFrontLeft.getTargetPosition() - this.motorFrontLeft.getCurrentPosition();
-        d = Math.min(d, this.motorFrontRight.getTargetPosition() - this.motorFrontRight.getCurrentPosition());
-        d = Math.min(d, this.motorBackLeft.getTargetPosition() - this.motorBackLeft.getCurrentPosition());
-        d = Math.min(d, this.motorBackRight.getTargetPosition() - this.motorBackRight.getCurrentPosition());
+        d = closestToZero(d, this.motorFrontRight.getTargetPosition() - this.motorFrontRight.getCurrentPosition());
```

Comment:
previously if d was negative, the return result was the lowest negative number which is the farthest away from zero.